### DISCUSSION TITLE
wc

### DISCUSSION THREAD

**Discussion by like112188** [Meloong-Git/PCL#4631] [提问] [answered]
打不开我的世界官网

**Comment by allMagicNB** [comment]
https://mc.163.com
https://minecraft.cn
https://minecraft.com.cn

**Comment by hejiehao** [comment]
截个图

**Comment by wyc-26** [comment, marked as answer]
挂 VPN 后再试一下……
应该是你的网络问题
我这不挂 VPN 仍可以正常访问
~以及你这标题不能换一下吗，你这样别人不知道你是来问啥问题的，来个 wc 还以为出什么大事了~

**Comment by 0LYT0** [comment]
6
